### ISSUE TITLE
Try other namenode(s) when one is in STANDBY

### ISSUE THREAD

**Issue by lancelet** [jystic/hadoop-rpc#5]
When a namenode is in a standby state, the `hh` tool produces the following error:
```
org.apache.hadoop.ipc.StandbyException
Operation category READ is not supported in state standby
```
(followed by a Java stacktrace)

This condition should presumably be detected and the command attempted on any alternate namenodes.

**Comment by jystic** [comment]
This particular problem is fixed in hh v0.7 ([the executable](https://github.com/jystic/hadoop-tools)) rather than here at the library level.

Users of the `hadoop-rpc` library have to deal with this problem themselves for the time being.

**Comment by jystic** [comment]
If anyone comes across this ticket and wants this feature, please make a new issue :)
